src/marlin_stubs/pause/M701_M702.cpp

```diff
@@ -40,17 +40,18 @@
 #include "marlin_server.hpp"
 #include "pause_stubbed.hpp"
 #include "filament.h"
+#include <functional>
 
 #define DO_NOT_RESTORE_Z_AXIS
 #define Z_AXIS_LOAD_POS   40
 #define Z_AXIS_UNLOAD_POS 20
 
-typedef void (*load_unload_fnc)(const int8_t target_extruder);
+using Func = std::function<void()>;
```

Comment:
Are you sure std function doesn't allocate memory? Not sure about current state, but there were times when std::function allocated memory to store the pointer. Please check the template definition thoroughly

Reply:
it is fine, it should pass pointer to anonymous functor (lambda)